src/index.ts

```diff
@@ -622,7 +623,19 @@ const initialize = (apiKey: string, target: Target, options?: Options): Result =
     eventSourceWithMiddleware = addMiddlewareToEventSource(middleware)
   }
 
-  return { on, off, variation, close, setEvaluations, registerAPIRequestMiddleware }
+  const refreshCache = (): boolean => {
+    // only fetch flags if enough time has elapsed to avoid pressuring backend servers
+    if (lastCacheRefreshTime == null || Math.round((Date.now()-lastCacheRefreshTime)/1000) >= 60) {
+      fetchFlags()
+      lastCacheRefreshTime = Date.now()
+      return true
+    }
+
+    logDebug('Refresh cache skipped. Not enough time has elapsed since last call')
```

Comment:
Again, `cache` is used in an unrelated way in this SDK and will be confusing for users.